lib/freyja/resource_factory.rb

```diff
@@ -131,17 +121,38 @@ def migrate_files!(resource:)
 
       ##
       # Move the given file to a backup directory, which is derived by injecting
-      # "backup-thumbnails" into the :path after the
+      # "backup-paths" into the :path after the
       # {Hyrax.config.derivatives_path} and before the other subdirectories.
       #
       # @param path [String]
-      def move_thumbnail_to_backup(path)
+      def move_derivative_to_backup(path)
         base_path = Hyrax.config.derivatives_path
         target_dirname = File.dirname(path).sub(base_path, File.join(base_path, "backup-paths"))
```

Comment:
I was getting a directory name like `backup-paths2e` when trying out the double_combo branch.
```suggestion
        target_dirname = File.dirname(path).sub(base_path, File.join(base_path, "backup-paths/"))
```